cli-plugins/apps/getting-started/configuration.hbs.md

```diff
@@ -0,0 +1,168 @@
+
+```bash
+export NO_COLOR=true
+```
+
+## .tanzuignore file
+
+There are many files and directories in projects that are not connected to running code
+(these files are not part of the final running container). When creating a workload from local
+source code, list these unused files and directories in the `.tanzuignore` file to avoid unnecessary
+consumption of resources when uploading the source.
+
+When iterating on code with the `--live-update` flag enabled, changes to directories or files
+listed in `.tanzuignore` do not trigger the automatic re-deployment of the source code.
+
+The following are some guidelines for the `.tanzuignore` file:
+
+- The `.tanzuignore` file should include a reference to itself, as it provides no value when deployed.
+- Directories must not end with the system separator `/`, or `\`.
+- Comments using hashtag `#` can be included.
+- If the `.tanzuignore` file contains files or directories that are not found in the source code, they are ignored.
+
+### Example of a .tanzuignore file
+
+```console
+.tanzuignore # must contain itself in order to be ignored
+# This is a comment
+this/is/a/folder/to/exclude
+
+this-is-a-file.ext
+```
+
+## Registry flags and environment variables
+
+The Apps CLI plug-in allows users to push images to their private registry by setting some flags 
+when creating workloads from local source code.
+
+A user can either trust a custom certificate on their system or pass the path to the certificate via flags.
+
+To pass the certificate through flags, specify:
+
+- `--registry-ca-cert`: This is the path of the self-signed certificate needed for the custom or 
+  private registry. This is also populated with a default value through the environment variable `TANZU_APPS_REGISTRY_CA_CERT`.
+- `--registry-password`: Use this when the registry requires credentials to push. The value of this 
+  flag can also be specified through `TANZU_APPS_REGISTRY_PASSWORD`.
+- `--registry-username`: Use with `--registry-password` to set the registry credentials. It can also
+  be provided as the environment variable `TANZU_APPS_REGISTRY_USERNAME`.
+- `--registry-token`: Set when the registry authentication is done through a token. The value of this
+  flag can also be taken from the `TANZU_APPS_REGISTRY_TOKEN` environment variable.
+
+For example:
+
+```bash
+tanzu apps workload apply WORKLOAD \
+--local-path PATH-TO-REPO -s registry.url.nip.io/PACKAGE/IMAGE \
+--type web --registry-ca-cert path/to/ca/cert.nip.io.crt \
+--registry-username USERNAME \
+--registry-password PASSWORD
+```
+
+Alternatively, the same command can be run as:
+
+```bash
+export TANZU_APPS_REGISTRY_CA_CERT=path/to/ca/cert.nip.io.crt
+export TANZU_APPS_REGISTRY_PASSWORD=USERNAME
+export TANZU_APPS_REGISTRY_USERNAME=PASSWORD
+
+tanzu apps workload apply WORKLOAD \
+--local-path PATH-TO-REPO \
+-s registry.url.nip.io/PACKAGE/IMAGE
+```
+
+`--type` is a flag that can be set through environment variables as well so it’s applied to all the
```

Comment:
`--type` is a flag which specifies the type of workload that's being created. The workload type value can be persisted across commands by setting the `TANZU_APPS_TYPE` environment variable.  The default value of `web` is set automatically if no `--type` flag or `TANZU_APPS_TYPE` value is provided.

Reply:
the above is meant for lines 143-145

Reply:
updated